client/v2/go.mod

```diff
@@ -12,48 +12,47 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
 	google.golang.org/grpc v1.52.3
-	google.golang.org/protobuf v1.28.1
+	google.golang.org/protobuf v1.28.2-0.20230130093322-0430d694e04a
```

Comment:
this is the bug with the script right?

Reply:
Nope, actually it isn't bad to update transitive dependency. 
But this precise case is because Comet upgraded to a pseudo version: https://github.com/cometbft/cometbft/blob/v0.37.x/go.mod#L57